cmd/argocd/commands/project.go

```diff
@@ -610,15 +608,37 @@ func modifyResourcesList(list *[]metav1.GroupKind, add bool, listDesc string, gr
 		}
 	}
 	if index == -1 {
-		fmt.Printf("Group '%s' and kind '%s' not in %s resources\n", group, kind, listDesc)
-		return false
+		return false, fmt.Sprintf("Group '%s' and kind '%s' not in %s resources", group, kind, listDesc)
 	}
 	*list = append((*list)[:index], (*list)[index+1:]...)
-	fmt.Printf("Group '%s' and kind '%s' is removed from %s resources\n", group, kind, listDesc)
-	return true
+	return true, fmt.Sprintf("Group '%s' and kind '%s' is removed from %s resources", group, kind, listDesc)
 }
 
-func modifyResourceListCmd(cmdUse, cmdDesc, examples string, clientOpts *argocdclient.ClientOptions, allow bool, namespacedList bool) *cobra.Command {
+func modifyAllowClusterResourceList(list *[]v1alpha1.ClusterResourceWhitelistItem, add bool, group string, kind string, name string) (bool, string) {
+	if add {
+		for _, item := range *list {
+			if item.Group == group && item.Kind == kind && (name == "" || item.Name == name) {
+				return false, fmt.Sprintf("Group '%s', kind '%s', and name '%s' is already present in cluster allow list", group, kind, name)
```

Comment:
Using `name=""` will be misleading for the user since they wont be able to know the current value of the name field.
```suggestion
				return false, fmt.Sprintf("Group '%s', kind '%s', and name '%s' is already present in cluster allow list", group, kind, item.name)
```